cms/envs/common.py

```diff
@@ -472,7 +473,7 @@
 
 PIPELINE_ENABLED = True
 
-STATICFILES_STORAGE = 'openedx.core.storage.ProductionStorage'
+STATICFILES_STORAGE = 'openedx.core.djangoapps.theming.storage.DevelopmentStorage'
```

Comment:
For security purposes, it's generally wiser to start with the strictest possible policy, and loosen it as necessary, rather than starting with a looser policy and tightening it as necessary. As a result, we generally default to production configurations of things, and loosen to development configurations when necessary. I don't think specific change is a security risk, but I don't want to encourage bad habits. If you feel strongly about this, I'll ask someone on the devops team to take a look at this and provide another opinion.

If your concern is people using `dev.py`, why not make the change in that file?